analyzers/src/SonarAnalyzer.Common/Rules/Utilities/SymbolReferenceAnalyzerBase.cs

```diff
@@ -137,8 +123,8 @@ protected sealed override SymbolReferenceInfo CreateMessage(SyntaxTree syntaxTre
             SymbolKind.TypeParameter
         };
 
-        private static SymRefInfo GetSymRefInfo(SyntaxToken token, SemanticModel semanticModel,
-            Func<SyntaxToken, bool> isIdentifier, Func<SyntaxToken, SyntaxNode> getBindableParent)
+        //FIXME: Deobfuscate
```

Comment:
I would remove these FIXME comments. We will do this anyway when we will need to change the code here.